src/azure-cli/requirements.py3.windows.txt

```diff
@@ -115,7 +115,7 @@ pypiwin32==223
 pyreadline==2.1
 python-dateutil==2.8.0
 pytz==2019.1
-pywin32==225
+pywin32==227
```

Comment:
Update `pywin32` to 227, otherwise `portalocker` will fail (https://github.com/mhammond/pywin32/issues/1431).